source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -56,6 +56,7 @@ To upgrade the All-In-One setup manually:
 *  Change to hass user `sudo su -s /bin/bash hass`
 *  Change to virtual enviroment `source /srv/hass/hass_venv/bin/activate`
 *  Update HA `pip3 install --upgrade homeassistant`
+*  Type `exit` to get out of hass user and change to normal user.
```

Comment:
Since your not changing to the `pi` user and only login out of the `hass` user I would prefer:
```
Type `exit` to logout the `hass` user and return to the `pi` user.
```